src/platform/src/main/java/org/geoserver/platform/resource/Paths.java

```diff
@@ -102,7 +102,41 @@ public static String path(String... path) {
         }
         return toPath(names);
     }
-
+    
+    // runtime flag which, if true, throws an error for the WARN characters
+    static final boolean STRICT_PATH = 
+            Boolean.valueOf(System.getProperty("STRICT_PATH", "true"));
```

Comment:
I've set it to false per your suggestion, although I know you and Jody have had a long discussion about how we will interact with the user. Perhaps it is okay to leave it like this for now, and I can re-open discussion on the mailing list for the appropriate way to have the user make this informed choice?